mule-extensions-api/src/main/java/org/mule/runtime/extension/internal/loader/ExtensionModelFactory.java

```diff
@@ -141,6 +143,7 @@
 public final class ExtensionModelFactory {
 
   public static final String PROBLEMS_HANDLER = "PROBLEMS_HANDLER";
+  public static final String COMPILATION_MODE_FLAG = "COMPILATION_MODE";
```

Comment:
shoudn't this go in `org.mule.runtime.api.util.MuleSystemProperties` alongside the other flags used in `ExtensionModelFactory`?